jwql/website/apps/jwql/monitor_pages/monitor_bias_bokeh.py

```diff
@@ -132,40 +125,23 @@ def retrieve_latest_data(self, aperture):
         aperture : str
             Aperture name (e.g. NRCA1_FULL)
         """
-        subq = (session.query(self.stats_table.aperture, func.max(self.stats_table.expstart).label("max_created"))
-                       .group_by(self.stats_table.aperture)
-                       .subquery()
-               )
-
-        query = (session.query(self.stats_table.aperture,
-                               self.stats_table.uncal_filename,
-                               self.stats_table.cal_filename,
-                               self.stats_table.cal_image,
-                               self.stats_table.expstart,
-                               self.stats_table.collapsed_rows,
-                               self.stats_table.collapsed_columns,
-                               self.stats_table.counts,
-                               self.stats_table.bin_centers,
-                               self.stats_table.entry_date)
-                        .filter(self.stats_table.aperture == aperture)
-                        .order_by(self.stats_table.entry_date) \
-                        .join(subq, self.stats_table.expstart == subq.c.max_created)
-                )
-
-        latest_data = query.all()
-        session.close()
-
-        # Put the returned data in a dataframe. Include only the most recent entry.
-        # The query has already filtered to include only entries using the latest
-        # expstart value.
-        self.latest_data = pd.DataFrame(latest_data[-1:], columns=['aperture', 'uncal_filename', 'cal_filename',
-                                                                   'cal_image', 'expstart_str', 'collapsed_rows',
-                                                                   'collapsed_columns', 'counts', 'bin_centers',
-                                                                   'entry_date'])
-        # Add a column of expstart values that are datetime objects
-        format_data = "%Y-%m-%dT%H:%M:%S.%f"
-        datetimes = [datetime.strptime(entry, format_data) for entry in self.latest_data['expstart_str']]
-        self.latest_data['expstart'] = datetimes
+        # Query database for the most recent bias stats entry with a matching aperture
+        columns = ['aperture', 'uncal_filename', 'cal_filename', 'cal_image', 'expstart',
+                   'collapsed_rows', 'collapsed_columns', 'counts', 'bin_centers', 'entry_date']
+        tmp_data = self.stats_table.objects.filter(aperture__iexact=aperture).order_by('-expstart').all().values(*columns).first()
+
+        # Put the returned data in a dataframe
+        if tmp_data is not None:
+            # Orient and transpose needed due to list column entries e.g. counts
+            self.latest_data = pd.DataFrame.from_dict(tmp_data, orient='index').transpose()
+
+            # Add a column of expstart values that are datetime objects
+            format_data = "%Y-%m-%dT%H:%M:%S.%f"
```

Comment:
Same here, if the log complains about a lack of timezone info

Reply:
No complaints in the log file about timezones. The expstart entries are strings rather than datetime objects.